static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-interview.scss

```diff
@@ -0,0 +1,12 @@
+@mixin overide-interview-headline-colours($pillar, $color1, $color2) {
```

Comment:
I like how this file was deleted yesterday, and then added again today 😆

Reply:
😅